kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeOutlinePanel.js

```diff
@@ -3,11 +3,12 @@
  * @author David Lyon  <dlyon@lbl.gov>
```

Comment:
mostly snake-> camel
also switched to using existing utils (see comments below)